tests/Integration/Behaviour/Features/Context/Domain/AliasFeatureContext.php

```diff
@@ -107,7 +106,7 @@ public function assertAliasesList(TableNode $table): void
     public function updateAliasStatus(bool $enable, string $aliasReference): void
     {
         $this->getCommandBus()->handle(new UpdateAliasStatusCommand(
-            new AliasId((int) $this->getSharedStorage()->get($aliasReference)),
+            (int) $this->getSharedStorage()->get($aliasReference),
```

Comment:
here as well, if the aliasId is saved to storage correctly during creation step, you shouldn't need to cast it everywhere

Reply:
Indeed, although t's not a major problem but it would be better without the casting